core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java

```diff
@@ -519,6 +519,7 @@ public RelNode findBestExp() {
         try {
           match.onMatch();
         } catch (VolcanoTimeoutException e) {
+          LOGGER.warn("Catch VolcanoTimeoutException, abort optimization.");
           root = canonize(root);
```

Comment:
How about "Volcano planning times out, cancels the subsequent optimization."

Reply:
> How about "Volcano planning times out, cancels the subsequent optimization."

Revised